formulaire/src/main/java/fr/openent/formulaire/controllers/QuestionController.java

```diff
@@ -142,54 +147,68 @@ public void create(HttpServerRequest request) {
                 log.error(message);
                 unauthorized(request, message);
                 return;
             }
 
-            RequestUtils.bodyToJson(request, question -> {
-                if (question == null || question.isEmpty()) {
-                    log.error("[Formulaire@QuestionController::createQuestion] No question to create.");
+            RequestUtils.bodyToJsonArray(request, questionsJson -> {
+                if (questionsJson == null || questionsJson.isEmpty()) {
+                    log.error("[Formulaire@QuestionController::create] No question to create.");
                     noContent(request);
                     return;
                 }
 
-                Long sectionId = question.getLong(SECTION_ID);
-                Long sectionPosition = question.getLong(SECTION_POSITION);
+                List<Question> questions = questionsJson.stream()
+                        .map(JsonObject.class::cast)
+                        .filter(Objects::nonNull)
+                        .map(question -> new Question(question))
+                        .filter(question -> question.getFormId() == Long.parseLong(formId))
+                        .collect(Collectors.toList());
 
                 // Check section infos validity
-                if (sectionId == null ^ sectionPosition == null) { // ^ = XOR -> return true if they have different value, return false if they both have same value
-                    String message = "[Formulaire@QuestionController::createQuestion] sectionId and sectionPosition must be both null or both not null.";
-                    log.error(message);
-                    badRequest(request);
-                    return;
-                } else if (sectionId != null && (sectionId <= 0 || sectionPosition <= 0)) {
-                    String message = "[Formulaire@QuestionController::createQuestion] sectionId and sectionPosition must have values greater than 0.";
-                    log.error(message);
-                    badRequest(request);
-                    return;
-                } else if (sectionPosition != null && question.getInteger(POSITION) != null) {
-                    String message = "[Formulaire@QuestionController::createQuestion] A question is either in or out of a " +
-                            "section, it cannot have a position and a section_position.";
-                    log.error(message);
-                    badRequest(request);
-                    return;
-                } else if (question.getBoolean(CONDITIONAL) && !CONDITIONAL_QUESTIONS.contains(question.getInteger(QUESTION_TYPE))) {
-                    String message = "[Formulaire@QuestionController::createQuestion] A question conditional question must be of type : " + CONDITIONAL_QUESTIONS;
-                    log.error(message);
-                    badRequest(request);
-                    return;
+                int i = 0;
+                while (i < questions.size()) {
```

Comment:
```suggestion
                for (Question question: questions)
```
ça évite de devoir géré un index